devito/operator.py

```diff
@@ -121,89 +122,56 @@ def __init__(self, expressions, **kwargs):
         # Track the DLE output, as it might be useful at execution time
         self._dle_state = dle_state
 
+        self.symbolic_data = [x for x in parameters if isinstance(x, SymbolicData)]
```

Comment:
I think these can be a property.

We should try to avoid class pollution as much as possible